scripts/integration/clickhouse/test.yaml

```diff
@@ -7,4 +7,4 @@ env:
   CLICKHOUSE_ADDRESS: http://clickhouse:8123
 
 matrix:
-  version: ['19']
+  version: ['23']
```

Comment:
Does this mean we no longer support back to 19, or just that we don't test that far back. I know we have a version requirement listed on the documentation page.